modules/API/API.js

```diff
@@ -463,6 +464,45 @@ function initCommands() {
             APP.store.dispatch(stopSharedVideo());
         },
 
+        /**
+         * Shows a custom in-meeting notification.
+         *
+         * @param { string } arg.title - Notification title.
+         * @param { string } arg.description - Notification description.
+         * @param { string } arg.type - Notification type, either `error`, `info`, `normal`, `success` or `warning`.
+         *                              Defaults to "normal" if not provided.
+         * @param { string } arg.timeout - Timeout type, either `short`, `medium`, `long` or `sticky`.
+         *                                 Defaults to "short" if not provided.
+         * @returns {void}
+         */
+        'show-notification': ({
```

Comment:
Could you please include an uid and a command to hide the notification?

Reply:
Will do.

Reply:
done :)